src/Compiler/AbstractIL/il.fs

```diff
@@ -3841,8 +3844,11 @@ let mkILGenericVirtualMethod (nm, access, genparams, actual_args, actual_ret, im
         body = notlazy impl
     )
 
-let mkILNonGenericVirtualMethod (nm, access, args, ret, impl) =
-    mkILGenericVirtualMethod (nm, access, mkILEmptyGenericParams, args, ret, impl)
+let mkILNonGenericVirtualMethod (nm, callconv, access, args, ret, impl) =
```

Comment:
Sanity check: non generic thing calls generic thing - is this correct?
Also, maybe worth adding "static" word to the name to distinguish from the new instance function.

Reply:
Yes, method was already there, we've added call conversion (instance vs non-instance).